src/main/java/core/basesyntax/RightTriangle.java

```diff
@@ -0,0 +1,24 @@
+package core.basesyntax;
+
+public class RightTriangle extends Figure {
+    private double firstLeg;
+    private double secondLeg;
+
+    public RightTriangle() {
+    }
+
+    public RightTriangle(double firstLeg, double secondLeg) {
+        this.typeOfFigure = TypeOfFigure.RIGHT_TRIANGLE;
+        this.area = getTriangleArea(firstLeg, secondLeg);
```

Comment:
Don't use static methods in your solution. The method `getTriangleArea` seems to be a static method that is being used to set the `area` field. It's better to create an instance method that can be called to calculate the area.